code/modules/clothing/outfits/standard.dm

```diff
@@ -113,6 +113,77 @@
 	head = /obj/item/clothing/head/helmet/bluetaghelm
 	shoes = /obj/item/clothing/shoes/sneakers/blue
 
+/datum/outfit/pirate
+	name = "Space Pirate"
+
+	id = /obj/item/card/id/advanced
+	id_trim = /datum/id_trim/pirate
+	uniform = /obj/item/clothing/under/costume/pirate
+	suit = /obj/item/clothing/suit/costume/pirate/armored
+	ears = /obj/item/radio/headset/syndicate
+	glasses = /obj/item/clothing/glasses/eyepatch
+	head = /obj/item/clothing/head/costume/pirate/bandana/armored
+	shoes = /obj/item/clothing/shoes/sneakers/brown
+
+/datum/outfit/pirate/post_equip(mob/living/carbon/human/equipped)
+	equipped.faction |= "pirate"
+
+	var/obj/item/radio/outfit_radio = equipped.ears
+	if(outfit_radio)
+		outfit_radio.set_frequency(FREQ_SYNDICATE)
+		outfit_radio.freqlock = RADIO_FREQENCY_LOCKED
+
+	var/obj/item/card/id/outfit_id = equipped.wear_id
+	if(outfit_id)
+		outfit_id.registered_name = equipped.real_name
+		outfit_id.update_label()
+		outfit_id.update_icon()
+
+	var/obj/item/clothing/under/pirate_uniform = equipped.w_uniform
+	if(pirate_uniform)
+		pirate_uniform.has_sensor = NO_SENSORS
+		pirate_uniform.sensor_mode = SENSOR_OFF
+		equipped.update_suit_sensors()
+
+/datum/outfit/pirate/captain
+	name = "Space Pirate Captain"
+
+	id_trim = /datum/id_trim/pirate/captain
+	head = /obj/item/clothing/head/costume/pirate/armored
+
+/datum/outfit/pirate/space
+	name = "Space Pirate (EVA)"
+
+	suit = /obj/item/clothing/suit/space/pirate
+	suit_store = /obj/item/tank/internals/oxygen
+	head = /obj/item/clothing/head/helmet/space/pirate/bandana
+	mask = /obj/item/clothing/mask/breath
+
+/datum/outfit/pirate/space/captain
+	name = "Space Pirate Captain (EVA)"
+
+	head = /obj/item/clothing/head/helmet/space/pirate
+
+/datum/outfit/pirate/silverscale
+	name = "Silver Scale Member"
+
+	id = /obj/item/card/id/advanced/silver
+	id_trim = /datum/id_trim/pirate/silverscale
+//	uniform = /obj/item/clothing/under/syndicate/sniper
```

Comment:
You could just make a new under with the same stats and sprites and leave it for the silverscales.

Reply:
ngl i forgot silverscales existed, and the fact that they used this suit.